src/sklearn_evaluation/plot/roc.py

```diff
@@ -291,7 +293,9 @@ def plot(self, ax=None):
         else:
             b_label = ["ROC curve 2"]
 
-        _generate_plot_from_fpr_tpr_lists(b.fpr, b.tpr, ax, label=b_label)
+        _generate_plot_from_fpr_tpr_lists(
+            b.fpr, b.tpr, ax, label=b_label, linestyle="dotted"
```

Comment:
is there an option to use dashes instead dots? i looked at some of the images and the dotted line is hard to see

update: I see our 45 degree line is dashed. so I think let's make the 45 degree one dotted and the other ones dashed and see how it looks.

Reply:
Everything should be fixed now and changelog updated